src/util/types.h

```diff
@@ -60,22 +61,6 @@ strToAssetCode(xdr::opaque_array<N>& ret, std::string const& str)
 bool addBalance(int64_t& balance, int64_t delta,
                 int64_t maxBalance = std::numeric_limits<int64_t>::max());
 
-enum Rounding
-{
-    ROUND_DOWN,
-    ROUND_UP
-};
-
-// calculates A*B/C when A*B overflows 64bits
-int64_t bigDivide(int64_t A, int64_t B, int64_t C, Rounding rounding);
-// no throw version, returns true if result is valid
-bool bigDivide(int64_t& result, int64_t A, int64_t B, int64_t C,
-               Rounding rounding);
-
-// no throw version, returns true if result is valid
-bool bigDivide(uint64_t& result, uint64_t A, uint64_t B, uint64_t C,
-               Rounding rounding);
-
```

Comment:
I think you should move the other related functions:
`bigDivide` and `bigMultiply`

Reply:
Right.
What about `addBalance` ? While the name is quite explicit about how it's used, the implementation is simplify an `add` with a ceiling.

Reply:
I'd leave it here as it's clearly a "stellar-core" type of function